### ISSUE TITLE
File template blah.0001 etc and gzip data; xia2 cannot "see"

### ISSUE THREAD

**Issue by graeme-winter** [xia2/xia2#184]
i.e.

```
Graemes-MacBook-Pro-5:run2 graeme$ xia2 min_images=3 image=/tmp/holton/MLYS_r1_1.0001.gz 
Environment configuration...
Python => /Users/graeme/svn/cctbx/build/../base/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
CCTBX => /Users/graeme/svn/cctbx/modules
CCP4 => /Applications/ccp4-7.0
CLIBD => /Applications/ccp4-7.0/lib/data
CCP4_SCR => /var/folders/hs/gvssfcd910s0jczjhjczj6h80000gn/T/tmpXqpOKj
Starting directory: /private/tmp/holton/run2
Working directory: /private/tmp/holton/run2
Free space:        53.67 GB
Contact: xia2.support@gmail.com
XIA2 0.5.441-g4def9d9b
DIALS 1.dev.1785-gc3447bb8
CCP4 7.0.042
Command line: xia2 min_images=3 image=/tmp/holton/MLYS_r1_1.0001.gz
Status: error "no sweeps found"
Please send the contents of xia2.txt, xia2.error and xia2-debug.txt to:
xia2.support@gmail.com
```

works fine if data not gzip...

**Comment by graeme-winter** [comment]
Fixed in 1d520ee13dc514141242073099ee30bcedbe52c9

**Comment by graeme-winter** [comment]
This seems fine - does not pick up e.g. 

```
-rw-r--r--  1 graeme  staff   6231506 28 Apr  2015 th_8_2sum_0106.cbf
-rw-r--r--  1 graeme  staff   6231482 28 Apr  2015 th_8_2sum_0105.cbf
-rw-r--r--  1 graeme  staff   6231358 28 Apr  2015 th_8_2sum_0104.cbf
-rw-r--r--  1 graeme  staff   6231596 28 Apr  2015 th_8_2sum_0103.cbf
-rw-r--r--  1 graeme  staff   6231528 28 Apr  2015 th_8_2sum_0102.cbf
-rw-r--r--  1 graeme  staff   6231388 28 Apr  2015 th_8_2sum_0108.cbf
-rw-r--r--  1 graeme  staff   6231470 28 Apr  2015 th_8_2sum_0107.cbf
-rw-r--r--@ 1 graeme  staff   3030225 25 Nov  2015 image0.png
-rw-r--r--@ 1 graeme  staff   3399757 20 Jan  2017 image0001.png
-rw-r--r--  1 graeme  staff  15191017 19 Oct 09:26 decoy.tar.gz
```
